### PR TITLE
Disable unused modules

### PR DESCRIPTION
**Pull request by Achal1607**
This PR aims to disable some obvious unused modules to reduce the size of the extension. After disabling all these modules the size of the extension reduces to almost 137 MB from almost 154 MB.

### CODE REVIEW

**Review by arvindaprameya — DISMISSED** [review]
Thank you, this will help with the reduction in the size of the extension

**Review by lahodaj — DISMISSED** [review]
Overall OK, I think. One note for the dark laf inline.

File: nbcode/nbproject/platform.properties [hunk 1]
```diff
@@ -316,6 +348,7 @@ disabled.modules=\
     org.netbeans.modules.xsl,\
     org.netbeans.spi.editor.hints.projects,\
     org.netbeans.swing.dirchooser,\
+    org.netbeans.swing.laf.dark,\
```
[the remark this answers is not in the document]
Comment (Achal1607) [reply nbcode/nbproject/platform.properties]: Thanks for pointing this out, I would remove it from the disabled modules list

**Review by arvindaprameya — APPROVED** [review]
LGTM, thanks

**Review by lahodaj — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**